src/error.rs

```diff
@@ -255,6 +290,28 @@ impl<Src, Dst: ?Sized> AlignmentError<Src, Dst> {
     }
 }
 
+impl<Src, Dst: ?Sized + Unaligned> AlignmentError<Src, Dst> {
+    fn into_infallible(self) -> Infallible {
```

Comment:
For consistency, should this be an implementation of `From`? Then we could have a consistent messaging that calling `.into()` at all levels makes impossible error branches `Infallible`.

Reply:
Done